azure/azure-pr-pipeline.yml

```diff
@@ -1,60 +1,42 @@
 name: "$(SourceBranchName)+$(BuildID)"
 
-# Prevents building every commit or on PR
 trigger: none
 pr: none
 
-# Consume an artifact from the build pipeline as soon as it becomes available.
 resources:
+  repositories:
+    - repository: common
+      type: github
+      name: NHSDigital/api-management-utils
+      ref: refs/heads/apm-1191-common-pipelines
```

Comment:
still got explicit ref to branch

Reply:
Removed for all pipelines now utils PR is merged